test/uq_test_parameters.toml

```diff
@@ -0,0 +1,82 @@
+# The six parameters below are to be used for calibration / uncertainty 
+# quantification
+[uq_param_1]
+prior = "Parameterized(Normal(2.0, 1.0))"
+constraint = "no_constraint()"
+type = "float"
+
+[uq_param_2]
+prior = "Parameterized(Gamma(5.0, 2.0))"
+constraint = "bounded_below(3.0)"
+type = "float"
+
+[uq_param_3]
+prior = "Parameterized(MvNormal(4, 0.1))"
```

Comment:
I see you do not have any value types here. are these ignored (I'd recommend this behaviour)

Reply:
I had added the "type" keys in analogy to the format of the "regular" CliMA parameters, but you are right that the information remains unused here, so I deleted all "type" information.